iOS_SDK/OneSignalSDK/Source/OneSignalCommonDefines.h

```diff
@@ -115,6 +121,7 @@
 #define IOS_USES_PROVISIONAL_AUTHORIZATION @"uses_provisional_auth"
 #define IOS_REQUIRES_EMAIL_AUTHENTICATION @"require_email_auth"
 #define IOS_RECEIVE_RECEIPTS_ENABLE @"receive_receipts_enable"
+#define IOS_OUTCOMES_V2_SERVICE_ENABLE @"outcomes_v2_service_enabled"
```

Comment:
This should be `v2_active` to match Android, and it should be under `outcomes`

Reply:
yeah I changed it on android (look to new PR) I though it looks more clear with this new name

Reply:
Let me know if you agree or if I should change it back

Reply:
Afte android change, I change it to v2_enabled